tests/security_features_test.go

```diff
@@ -141,6 +142,30 @@ var _ = Describe("SecurityFeatures", func() {
 				}
 				Expect(*container.SecurityContext.Privileged).To(BeFalse())
 			})
+
+			It("[test_id:5297]Make sure qemu processes are MCS constrained", func() {
+
+				By("Starting a VirtualMachineInstance")
+				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				tests.WaitForSuccessfulVMIStart(vmi)
+
+				pod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
+
+				qemuProcess, err := tests.ExecuteCommandOnPod(
+					virtClient,
+					pod,
+					"compute",
+					[]string{"/usr/bin/bash", "-c", "ps -efZ | grep [q]emu-kvm | awk '{print $1}'"},
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Checking that qemu-kvm process is of the SELinux type container_t")
+				Expect(strings.Split(qemuProcess, ":")[2]).To(Equal("container_t"))
+
+				By("Checking that qemu-kvm process has SELinux category_set")
+				Expect(len(strings.Split(qemuProcess, ":"))).To(Equal(5))
```

Comment:
The category_set is random and so not sure, what to compare it with.
Hence decided to check for it's length, if category_set exists the length would be 5, ifnot then 4.

Reply:
:+1: